### ISSUE TITLE
New notification types aren't read by screenreader when focused

### ISSUE THREAD

**Issue by ObjectInSpace** [enafore/enafore#32]
I use Chrome + the NVDA screenreader to access the web. I usually navigate by putting it in "focus mode" which sends input directly to the browser. I'll focus the list and use up and down arrows to traverse. When I do this in the notifications list, it will read mentions, boosts, favorites and follows. However it will not read when someone you follow has migrated to a new instance. It also won't read the new grouped notifications. Instead it just says "document."

Please fix!

**Comment by rkingett** [comment]
I can reproduce this on Firefox, as well.

**Comment by easrng** [comment]
thanks for reporting this! looking into it now.

**Comment by easrng** [comment]
I'm disabling grouping for now while I fix up notification handling more generally. Right now information about how the various types are handled is scattered all over the codebase so I want to fix that.

**Comment by easrng** [comment]
This should be fixed now + I added accessible names for the notification types that didn't have them!

**Comment by rkingett** [comment]
After this update, there is constant verbosity that says, unhandled notification type, before the notification type. The type, of course, changes depending on the notofication type like follows and similar but on every notification, the screen reader is announcing unhandled.

**Comment by easrng** [comment]
aaaaa sorry. will fix tonight. all notifications or just mentions/subscribed posts?

**Comment by ObjectInSpace** [comment]
I'm just seeing it for mentions so far.

**Comment by easrng** [comment]
should be fixed.
